pkgs/tools/networking/n2n/default.nix

```
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub, autoconf, automake, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "n2n";
+  version = "2.8";
+
+  src = fetchFromGitHub {
+    owner = "ntop";
+    repo = "n2n";
+    rev = version;
+    hash = "sha256-2xJ8gYVZJZoKs6PZ/9GacgxQ+/3tmnRntT1AbPe1At4=";
+  };
+
+  nativeBuildInputs = [ autoconf automake pkg-config ];
+
+  preConfigure = ''
+    patchShebangs autogen.sh
+    ./autogen.sh
+  '';
+
+  # According to the README, some parts of the code significantly benefit from
+  # compiler optimizations and platform features such as NEON, SSE and AVX.
+  preferLocalBuild = true;
+  CFLAGS="-O3 -march=native";
+
+  installPhase = ''
+    make PREFIX=$out install
+  '';
```

`make install` is ran by default in the `installPhase`, we only need to set the prefix correctly.
```suggestion
  PREFIX = placeholder "out";
```
Alternatively, `makeFlags` could be used.